examples/scales/logit_demo.py

```diff
@@ -0,0 +1,57 @@
+"""
+================
+Logit Demo
+================
+
+Examples of plots with logit axes.
+"""
+
+import numpy as np
+import scipy as sp
+import scipy.stats
```

Comment:
right now the docs build independently of scipy; it's not really worth adding scipy as a dependency just for this (you can implement whatever cdf formula you want inline?)

Reply:
I cant. For inline this I need the hypergeometric function which is in scipy.special.

But it is only a example, I can change the distribution used in example.

Reply:
please do

Reply:
Done. I choose simpler distributions. I was prefering using students distrib, but it is only because it is my self usage of the logit scale.